action.yml

```diff
@@ -2,6 +2,9 @@ name: 'Setup Node.js environment'
 description: 'Setup a Node.js environment and add it to the PATH, additionally providing proxy support'
 author: 'GitHub'
 inputs:
+  always-auth:
```

Comment:
Nit - this should go after scope in the parameter ordering. That way its grouped with the other auth parameters and comes after registry-url which is required in all auth cases.

Reply:
Counter nit: Opinion: At the top it leaves the map in alphabetical order.